deploy/template.yaml

```diff
@@ -2124,6 +2143,17 @@ Resources:
               - ReportBatchItemFailures
       AutoPublishAlias: live
 
+  StagingAsyncCriResponseMapping:
```

Comment:
Could this be configured in`Events` above like the original mapping, probably as a new event?